VERSION

```diff
@@ -0,0 +1 @@
+1.0.0-pre1
```

Comment:
What would we use this file for? Could we replace it with `git describe HEAD`? (or `git describe --abbrev=0 HEAD` if we don't want the exact commit)

When we're deploying production `vector-datasource`, the Chef recipes point to a specific tag, so the `VERSION` file is superfluous for the purposes of identifying the code version and it could potentially be in conflict with the tag - which would be right?

If we want to keep the `VERSION` file, and the version in `setup.py` as well, should we add a lint / CI check to ensure that the version matches the git tag?